# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# T. Tran <transifex@emiu.net>, 2020
# th1nhhdk <th1nhhdk@tutanota.com>, 2021
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-28 22:49+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: th1nhhdk <th1nhhdk@tutanota.com>, 2021\n"
"Language-Team: Vietnamese (https://app.transifex.com/calamares/teams/20061/vi/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: vi\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "Đang cài đặt bộ khởi động."

#: src/modules/bootloader/main.py:644
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""

#: src/modules/bootloader/main.py:899
msgid "Bootloader installation error"
msgstr "Lỗi cài đặt trình khởi động(bootloader)"

#: src/modules/bootloader/main.py:900
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Trình khởi động(bootloader) không thể được cài đặt. Lệnh cài đặt "
"<pre>{!s}</pre>đã trả mã lỗi {!s}."

#: src/modules/displaymanager/main.py:509
msgid "Cannot write LXDM configuration file"
msgstr "Không thể ghi vào tập tin cấu hình LXDM"

#: src/modules/displaymanager/main.py:510
msgid "LXDM config file {!s} does not exist"
msgstr "Tập tin cấu hình LXDM {!s} không tồn tại"

#: src/modules/displaymanager/main.py:598
msgid "Cannot write LightDM configuration file"
msgstr "Không thể ghi vào tập tin cấu hình LightDM"

#: src/modules/displaymanager/main.py:599
msgid "LightDM config file {!s} does not exist"
msgstr "Tập tin cấu hình LightDM {!s} không tồn tại"

#: src/modules/displaymanager/main.py:684
msgid "Cannot configure LightDM"
msgstr "Không thể cấu hình LXDM"

#: src/modules/displaymanager/main.py:685
msgid "No LightDM greeter installed."
msgstr "Màn hình chào mừng LightDM không được cài đặt."

#: src/modules/displaymanager/main.py:716
msgid "Cannot write SLIM configuration file"
msgstr "Không thể ghi vào tập tin cấu hình SLIM"

#: src/modules/displaymanager/main.py:717
msgid "SLIM config file {!s} does not exist"
msgstr "Tập tin cấu hình SLIM {!s} không tồn tại"

#: src/modules/displaymanager/main.py:935
msgid "No display managers selected for the displaymanager module."
msgstr ""
"Không có trình quản lý hiển thị nào được chọn cho mô-đun quản lý hiển thị"

#: src/modules/displaymanager/main.py:936
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"Danh sách quản lý hiện thị trống hoặc không được định nghĩa cả trong "
"globalstorage và displaymanager.conf."

#: src/modules/displaymanager/main.py:1023
msgid "Display manager configuration was incomplete"
msgstr "Cầu hình quản lý hiện thị không hoàn tất"

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Đang tạo initramfs bằng dracut."

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr ""

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Ví dụ công việc python."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Ví dụ python bước {}"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "Đang viết vào fstab."

#: src/modules/fstab/main.py:377 src/modules/fstab/main.py:383
#: src/modules/fstab/main.py:411 src/modules/initcpiocfg/main.py:256
#: src/modules/initcpiocfg/main.py:260 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:329 src/modules/networkcfg/main.py:105
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Lỗi cấu hình"

#: src/modules/fstab/main.py:378 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:330 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "Không có phân vùng nào được định nghĩa cho <pre>{!s}</pre> để dùng."

#: src/modules/fstab/main.py:384 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr "Không có điểm kết nối gốc cho <pre>{!s}</pre> để dùng."

#: src/modules/fstab/main.py:412
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/grubcfg/main.py:29
msgid "Configure GRUB."
msgstr "Cấu hình GRUB"

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Đang thiết lập đồng hồ máy tính."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Đang cấu hình mkinitcpio."

#: src/modules/initcpiocfg/main.py:257
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr ""

#: src/modules/initcpiocfg/main.py:261
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr ""

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Đang cấu hình initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Đang cấu hình ngôn ngữ."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "Đang tạo initramfs bằng mkinitfs."

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "Chạy mkinitfs thất bại ở hệ thống đích"

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "Mã lỗi trả về là {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Đang gắn kết các phân vùng."

#: src/modules/mount/main.py:164 src/modules/mount/main.py:200
msgid "Internal error mounting zfs datasets"
msgstr ""

#: src/modules/mount/main.py:176
msgid "Failed to import zpool"
msgstr ""

#: src/modules/mount/main.py:192
msgid "Failed to unlock zpool"
msgstr ""

#: src/modules/mount/main.py:209 src/modules/mount/main.py:214
msgid "Failed to set zfs mountpoint"
msgstr ""

#: src/modules/mount/main.py:365
msgid "zfs mounting error"
msgstr ""

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Đang lưu cấu hình mạng."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "Đang cấu hình dịch vụ OpenRC dmcrypt."

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Đang cài đặt các gói ứng dụng."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Đang xử lý gói (%(count)d / %(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Đang cài đặt %(num)d gói ứng dụng."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Đang gỡ bỏ %(num)d gói ứng dụng."

#: src/modules/packages/main.py:740 src/modules/packages/main.py:752
#: src/modules/packages/main.py:780
msgid "Package Manager error"
msgstr ""

#: src/modules/packages/main.py:741
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:753
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:781
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Cấu hình giao diện Plymouth"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Đang cài đặt dữ liệu."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Cấu hình dịch vụ OpenRC"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "Không thể thêm dịch vụ {name!s} vào run-level {level!s}."

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr "Không thể loại bỏ dịch vụ {name!s} từ run-level {level!s}."

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"Không nhận ra thao tác <code>{arg!s}</code> cho dịch vụ {name!s} ở run-level"
" {level!s}."

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "Không thể sửa đổi dịch vụ"

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"Lệnh <code>rc-update {arg!s}</code> trong môi trường chroot trả về lỗi "
"{num!s}."

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "Nhóm dịch vụ khởi động không tồn tại"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Đường dẫn cho runlevel {level!s} là <code>{path!s}</code>, nhưng không tồn "
"tại."

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "Nhóm dịch vụ không tồn tại"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Đường dẫn cho dịch vụ {name!s} là <code>{path!s}</code>, nhưng không tồn "
"tại."

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr ""

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr ""

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr ""

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Đang làm đầy các hệ thống tập tin."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "rsync thất bại với lỗi {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Đang bung hình ảnh {}/{}, tập tin {}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Bắt đầu bung nội dung {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "Bung hình ảnh thất bại \"{}\""

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "Không có điểm kết nối cho phân vùng gốc"

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr ""

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Sai điểm kết nối cho phân vùng gốc"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr ""

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr ""

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr ""

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr "Hệ thống tập tin cho \"{}\" ({}) không được hỗ trợ bởi nhân hiện tại"

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "Hệ thống tập tin nguồn \"{}\" không tồn tại"

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "Hệ thống đích \"{}\" không phải là một thư mục"

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr ""
